### PR TITLE
Codeandbox icon addition

### PR DESCRIPTION
**Pull request by LucasZapico**
**Issue:** #1183 (#1184)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by LucasZapico** [comment]
Hey @ericcornelissen, 

Thanks for your help. This is my first open source pull request 💪🤓. 

Firstly, to remedy the 39 commits issue 😝. I referenced the rebase articles you linked, but was not confident in my understanding of some of the issues associated with rebasing. Rather, I reverted my dev and codesandbox-icon-addition branches back to the status at #1177. Then ```cherry-picked``` my commits from a tmp branch into my new codesandbox-icon-addition branch. Then blew away and re-pushed my remote codesandbox-icon-addition and did the PR. I hope at the very least this works if not the most direct method. 

It was a solid review of git btw. 👍

If it is not too much trouble, please let me know if there is anything in particular that I should do differently as I would like to make my contributions as seamless as possible for simple-icons and my future in open source. Hopefully a long and fruitful relationship. 😇

File: icons/codesandbox.svg [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<svg viewBox="0 0 24 24" rule="img" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 24 24" role="img" xmlns="http://www.w3.org/2000/svg">
```
Comment (pzygielo) [inline icons/codesandbox.svg]: See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon:
> Be minified to a single line with no formatting.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -389,6 +389,11 @@
             "title": "Coderwall",
             "hex": "3E8DCC",
             "source": "https://github.com/twolfson/coderwall-svg"
+        },
+         {
```
Comment (pzygielo) [inline _data/simple-icons.json]: One space too much.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -389,6 +389,11 @@
...
+          "title": "CodeSandbox",
```
Comment (pzygielo) [inline _data/simple-icons.json]: Older elements looks to be two spaces to the right.

**Comment by ericcornelissen** [comment]
I'm happy to help @LucasZapico, we aim for Simple Icons to be a nice place for first time contributors 😄 Your solution for the 39 commits is just fine 👍 Now just fix the comments by @pzygielo and I think we're good to go.

I think you got contribution to simple-icons down after this PR. For the future make sure to use the [Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) as reference, that also goes for other open source projects. It can be a bit much to read at times, but it does make your contributions more seamless 😉

**Comment by LucasZapico** [comment]
@ericcornelissen Thanks for the welcome.

 I promise I read and have been referencing the [Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md). I guess I'm suffering from first open source PR jitters. 😣 I hope I corrected and caught everything this time. 

Thank you for your patients. 🙏

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.13",
+  "version": "1.9.19",
```
Comment (pzygielo) [inline package.json]: I think maintainers will take care for version update upon release.
Could you restore this value, and the one in lock file? We'll end up with one new file and one modified only then.

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.19",
+  "version": "1.9.13",
```
Comment (pzygielo) [inline package.json]: Set it to 1.9.19, as in current develop in origin repository. 
Commit only this one change and push.

Comment (pzygielo) [reply package.json]: There are multiple ways to resolve conflict with this file.

As in your work you changed `package.json`, and in the mean time it was also changed on the branch you want to merge to - possibilities are, not limited to:

1. set this line to the value that *already is in develop* (i.e. "version": "1.9.19") (will be discovered as no change to merge)
2. restore it to the value *you started with*, and currently your branch starts at a853feb, and the line is "version": "1.9.18" there (will be discovered as no change done by you).

There are other solutions but I think _1_ is the easiest.

File: package.json (hunk 4 above)
Comment (ericcornelissen) [reply package.json]: In this regard no, I cannot remember an instance where a contributor changed the `"version"` in the package.json so we've never considered it. That being said, I think the Contributing Guidelines are clear on what you **should** change (in the case you're adding/updating an icon) and other than that contributors **shouldn't** change anything.

But don't worry about having changed it, that's what PR reviews are for 😃

**Comment by ericcornelissen** [comment]
I trust you read and reference the Contributing Guidelines @LucasZapico 😉 it was just a piece of advise. Also, one might interpret the contributing guidelines different from how it was intended and that is not a problem. In my experience most open source maintainers are not that harsh if you mess up 👍 

You're almost there now, just make sure the version number in package.json is the same as the one on the `develop` branch so there are no merge conflicts.

File: package.json (hunk 5 above)
Comment (LucasZapico) [reply package.json]: Awesome! I will change it to version 1.9.19. Thank you for the explanation. 🙏

**Comment by LucasZapico** [comment]
@ericcornelissen, 

Again, thanks for the guidance. Now I'll get the SketchApp icon done and I'll look into contributing to the Website redesign.

**Comment by ericcornelissen** [comment]
Thanks for you contribution @LucasZapico and thanks for the reviews @pzygielo 🎉 

As for the SketchApp logo, have a quick look at the discussion in #788 😉

**Pull request merged**